### PR TITLE
Update service connection name to fix builds

### PR DESCRIPTION
**Pull request by kichalla**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs [hunk 1]
```diff
@@ -1269,6 +1269,54 @@ public void BuildsAppSuccessfully_EvenIfRequirementsTxtOrSetupPyFileDoNotExist()
...
+            var script = new ShellScriptBuilder()
+                .AddDefaultTestEnvironmentVariables()
+                .AddCommand(
+                $"oryx build {appDir} -i {buildDir} -o {outputDir} " +
+                $"-p virtualenv_name={virtualEnvName} --compress-destination-dir")
+                .AddFileExistsCheck($"{outputDir}/output.tar.gz")
+                .AddFileExistsCheck($"{outputDir}/oryx-manifest.toml")
+                .AddFileDoesNotExistCheck($"{outputDir}/requirements.txt")
+                .AddDirectoryDoesNotExistCheck($"{outputDir}/{virtualEnvName}")
+                .AddCommand($"cat {manifestFile}")
```
Comment (arroyc) [inline tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs:1292]: nit: remove cat command ?

File: src/BuildScriptGeneratorCli/Commands/BuildCommandBase.cs [hunk 2]
```diff
@@ -39,6 +39,12 @@ internal abstract class BuildCommandBase : CommandBase
             Description = "Type of application that the the source directory has, for example: 'functions' or 'static-sites' etc.")]
         public string AppType { get; set; }
 
+        [Option(
+            OptionTemplates.CompressDestinationDir,
+            CommandOptionType.NoValue,
+            Description = "Compresses the destination directory(excluding the manifest file) into a tar ball.")]
```
Comment (arroyc) [inline src/BuildScriptGeneratorCli/Commands/BuildCommandBase.cs:45]: curious: why dont we compress the manifest as well?

**Review by arroyc — COMMENTED** [review]
can you trigger a nightly build as well?

**Review by arroyc — APPROVED** [review]

**Comment by kichalla** [comment]
Closing this PR as we do not want to check this in into `master` branch. We will get a build out of the `exp/` branches.

**Pull request merged**